lib/opensusebasetest.pm

```diff
@@ -1049,13 +1049,22 @@ sub wait_boot_past_bootloader {
     my $nologin = $args{nologin};
     my $forcenologin = $args{forcenologin};
 
-    # Workaround for bsc#1204221 and bsc#1204230
-    if (is_sle('=15-SP5') && check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
-        # This should only happen on SLE15SP5 and on hyperv
-        record_soft_failure 'workaround bsc#1204221 - Failed to boot at SLES15SP5 with gnome in hyperv 2019' if check_var('DESKTOP', 'gnome');
-        record_soft_failure 'workaround bsc#1204230 - Failed to boot at SLES15SP5 with x server in hyperv 2019' if check_var('DESKTOP', 'textmode');
-        sleep 30;
-        send_key 'esc';
+    # Workaround for bsc#1204221, bsc#1204230 and bsc#1203641
+    if (is_sle('=15-SP5') && check_var('VIRSH_VMM_FAMILY', 'hyperv') && check_var('HYPERV_VERSION', '2019') && check_var('FLAVOR', 'Online')) {
```

Comment:
Why you need this ?
```
check_var('FLAVOR', 'Online')
```

Reply:
It should not include workaround for jobs: [snapper_jeos_cli/36](https://openqa.suse.de/tests/9823700#step/snapper_jeos_cli/36)
so filter it.